scripts/gen_single_image_diffusion.py

```diff
@@ -48,14 +48,14 @@
 
 
 def load_model(
-    modelpath,
-    model_in_file,
+    model_in_dir,
```

Comment:
`model_in_dir`  usually points to the pth file, it doesn't seem this is appropriate here ?